bandit/src/get-bandit-tests/dynamo.ts

```diff
@@ -13,10 +13,9 @@ const runQuery =(
 			ExpressionAttributeValues: {
 				":channel":channel,
 				":draft": "Draft",
-				":isBanditTest": true,
```

Comment:
Just curious why we are removing this and not adding in a bandit filed here?

Reply:
Previously we were asking Dynamodb to do the filtering of bandit tests, by checking the `isBanditTest` boolean field.
But now that I've replaced that with a more complex model (an array of objects), we can't do the filtering here. So instead we ask Dynamodb for all live tests, and do the filtering after